### PR TITLE
Added security policy

### PR DESCRIPTION
**Pull request by melpadden**
Adds an issue template and a SECURITY.md file

### CODE REVIEW

**Comment by sacherjj** [comment]
Supported versions will be outdated.  We should probably push this after we make release-1.5.7 public and update to have 1.5.7 as this version.

File: .github/ISSUE_TEMPLATE/security_vulnerability.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This is a submission form for Casper security issues.
+  - type: input
+    id: email
+    attributes:
+      label: Your email address
+      placeholder: john@doe.com
```
Comment (sacherjj) [inline .github/ISSUE_TEMPLATE/security_vulnerability.yml:15]: Will john@doe.com forward requests to us?

**Review by mpapierski — CHANGES_REQUESTED** [review]
I have some concerns

**Comment by mpapierski** [comment]
Closing as discussed on standup

**Pull request closed without merging**